app/src/main/res/layout/player.xml

```diff
@@ -186,6 +197,7 @@
         android:textStyle="bold"
         android:visibility="gone" />
 
+    <!--
```

Comment:
The top controls should definitely be kept.
I suggest making the menu button selectable by reserving e.g. the last index for it. Then it doesn't need to be moved to the control group.

Reply:
Done.
Looked good in the control bar though, but it's probably better to have a separate menu.